<launch>
    <node pkg="path_planning" type="mgeo_pub.py" name="mgeo_pub"/>

    <node pkg="path_planning" type="gpsimu_parser.py" name="GPS_IMU_parser"/>
    <node pkg="path_planning" type="tf_pub.py" name="tf"/>

    <node pkg="path_planning" type="global_path_pub.py" name="global_path"/>
    <node pkg="path_planning" type="local_path_pub.py" name="local_path"/>

    <node pkg="prediction" type="object_tracker.py" name="tracker_dynamic_obstacle"/>
    <node pkg="prediction" type="object_collision.py" name="collision_detection" output="screen"/>
    <node pkg="prediction" type="tracker_visualize.py" name="visualize_dynamic_obstacle"/>

    <node pkg="control" type="pure_pursuit.py" name="pure_pursuit" output="screen"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find prediction)/rviz/visual_test.rviz"/>
</launch>
